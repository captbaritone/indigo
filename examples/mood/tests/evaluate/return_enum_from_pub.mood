enum Boolean {
  True,
  False
}

pub fn test(): Boolean {
  Boolean::True
}