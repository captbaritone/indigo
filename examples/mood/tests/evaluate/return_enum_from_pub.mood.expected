-----------------
INPUT
----------------- 
enum Boolean {
  True,
  False
}

pub fn test(): Boolean {
  Boolean::True
}
-----------------
OUTPUT
-----------------
0